docs/source/quickstart.rst

```diff
@@ -1,9 +1,49 @@
 Quickstart
 ==========
 
-Run the main code via:
-```
-python jfsd/JFSD.py
-```
+## How to Run Simulations
 
-During the simulation, the particles trajectories,velocities, and stresslets are saved in a numpy array of shape (N_s, N_p, N_c), with N_s the number of frames stored, N_p the number of particles and N_c the number of d.o.f. (3 for trajectories, 6 for velocities, 5 for stresslets).
+1 - Create a configuration .toml in files/ (use "files/example_configuration.toml" as reference)
+
+2 - Run the main code, from the project directory, via:
+
+.. code-block:: shell
+
+	jfsd -c files/<your_config_file.toml> -o <directory_name_for_output>
```

Comment:
I would personally use the long format here `--config` and `--output` (please check if those are the actual correct flags), since this make easier to read if slightly more verbose.